integration-libs/cdc/manage-users/manage-users-list/cdc-list.component.html

```diff
@@ -0,0 +1,96 @@
+<!--copied from core module and altered create button attributes-->
```

Comment:
In this PR we copy a lot of code from core libraries in order to introduce small CDC changes. I hope there should exist a more optimal way. Copying big portions of code is harder to maintain (usually when adding improvements in the core lib, the copied version is forgotten to be updated).

Did you consider any alternatives instead of copying whole html templates? and instead of copying the whole tree of CMS child routes? 

From top of my head (not sure if applicable well in your case):
- for changing TS logic: we can introduce an extendable service in the core lib, and then extend it in CDC lib (similar to what we did in previous [CDC feature login/register](https://github.com/SAP/spartacus/pull/15785#discussion_r884857549) )
- for changing the UI: introduce an UI outlet in the core UI, and then populate it in the CDC lib (the price is not loading your CDC UI addition, but if it's fairly small we could accept this tradeoff)

Thanks to it I hope we could potentially avoid replacing the whole components via the CMS configuration (especially needing to copy-paste the whole tree of child routes).

What do you think about it?

----
btw. the copied UI is already dis-synchronized with the core UI (which have been changed recently to improve a11y. That's why I want to avoid copy-pasting big portions of HTML, if only possible).

Reply:
Hi Kris, I have updated the code for 'Add'/'Manage Users' functionality in this commit [76d47b2](https://github.com/SAP/spartacus/pull/16748/commits/76d47b20405bebce0aeb4a9ea6e30595d25ceba3) .. can you review it and provide your feedback.

Reply:
@Platonn I have also changed the code for 'user details' screen as well in [c54b4d8](https://github.com/SAP/spartacus/pull/16748/commits/c54b4d80ea34cc2aeb5727e92f82c070f7bb3928) please review and give your feedback